### DISCUSSION TITLE
Misplaced objects

### DISCUSSION THREAD

**Discussion by amelillo** [senshu/Sozi#575] [Q&A]
Hello.
I have some Sozi presentation in which I carefully plan every move.
In Sozi I see them working perfectly.
Though, when I look at them in a browser, some of the objects are misplaced and enlarged, so the figures are all messed up.
I can't figure out what's wrong.
Here they are.
Can anyone help me?
[PO_prisma.zip](https://github.com/sozi-projects/Sozi/files/7610210/PO_prisma.zip)

**Comment by amelillo** [comment]
Moreover, I tried rendering it in Firefox and Chrome, both under Linux, same result.
I noticed that if I save the first four or five frames the object are placed exactly where they're supposed to be.
Adding a step or simply opening the svg again in Sozi (thus causing an overwrite of the existing html files) corrupts the presentation which now shows misplaced lines even if I don't change anything.
I'm pretty puzzled.

**Reply by hoshi411** [reply]
can't open the file now but have you tried converting the inkscape objects which are acting odd, to paths?

**Reply by amelillo** [reply]
Sure I did, but if you follow recent updates, the last February release should solve the bug.
There's a workaround too for earlier versions: adding a transparent rectangular shape helps Sozi addressing objects and their correct size.
Cheers!
